### ISSUE TITLE
Issues with LedgerKeyXDR parsing.

### ISSUE THREAD

**Issue by light-cloud** [Soneso/stellar-ios-mac-sdk#123]
**1. Parsing ClaimableBalanceIDXDR**
Can’t parse `ClaimableBalanceIDXDR` inside `LedgerKeyXDR` enum, because of typo inside `init` .
For claimable balance should be `case LedgerEntryType.claimableBalance.rawValue` instead of `LedgerEntryType.data.rawValue`.

```
    public init(from decoder: Decoder) throws {
        var container = try decoder.unkeyedContainer()
        
        let type = try container.decode(Int32.self)
        
        switch type {
        case LedgerEntryType.account.rawValue:
            let acc = try container.decode(LedgerKeyAccountXDR.self)
            self = .account(acc)
        case LedgerEntryType.trustline.rawValue:
            let trus = try container.decode(LedgerKeyTrustLineXDR.self)
            self = .trustline(trus)
        case LedgerEntryType.offer.rawValue:
            let offeru = try container.decode(LedgerKeyOfferXDR.self)
            self = .offer(offeru)
        case LedgerEntryType.data.rawValue:
            let datamu = try container.decode(LedgerKeyDataXDR.self)
            self = .data (datamu)
        case LedgerEntryType.data.rawValue:
            let value = try container.decode(ClaimableBalanceIDXDR.self)
            self = .claimableBalance (value)
        default:
            let acc = try container.decode(LedgerKeyAccountXDR.self)
            self = .account(acc)
        }
    }

```

**2. Not possible to display detailed info about LedgerKeyXDR to user**

```
public enum LedgerKeyXDR: XDRCodable {
    case account (LedgerKeyAccountXDR)
    case trustline (LedgerKeyTrustLineXDR)
    case offer (LedgerKeyOfferXDR)
    case data (LedgerKeyDataXDR)
    case claimableBalance (ClaimableBalanceIDXDR)
```

To access this data we need, to make these properties `public`, but they are `internal` now. Please, check, for example, `LedgerKeyAccountXDR` and update `LedgerKeyTrustLineXDR`, `LedgerKeyOfferXDR`, `LedgerKeyDataXDR`.
```
public struct LedgerKeyTrustLineXDR: XDRCodable {
    let accountID: PublicKey
    let asset: AssetXDR
}

```
```
public struct LedgerKeyOfferXDR: XDRCodable {
    let sellerId: PublicKey
    let offerId: UInt64

}

```

```
public struct LedgerKeyDataXDR: XDRCodable {
    let accountId: PublicKey
    let dataName: String
}
```

**Comment by christian-rogobete** [comment]
Hello @light-cloud, thank you for adding this issue. Are you going to fix the 2 problems or should I do it?

**Comment by light-cloud** [comment]
Hello @christian-rogobete. Please, review https://github.com/Soneso/stellar-ios-mac-sdk/pull/125.

**Comment by christian-rogobete** [comment]
resolved in [release 2.0.3](https://github.com/Soneso/stellar-ios-mac-sdk/releases/tag/2.0.3)
